playbook/app/pb_kits/playbook/pb_multi_level_select/_multi_level_select.tsx

```diff
@@ -78,6 +84,11 @@ const MultiLevelSelect = (props: MultiLevelSelectProps) => {
   useEffect(() => {
     if (returnAllSelected) {
       setReturnedArray(getCheckedItems(formattedData))
+    } else if (variant === "single" && selectedIds?.length) {
+      // The default is the first selectedId if there are more than one
+      const defaultSelection = filterFormattedDataById(formattedData, selectedIds[0])
+      setSingleSelectInputValue(defaultSelection[0]?.value)
+      setDefaultReturn(defaultSelection)
```

Comment:
Here we prevent multiple default options and populate the text input with the default option.